endpoint.go

```diff
@@ -638,7 +638,24 @@ func (ep *endpoint) assignAddress() error {
 	if err != nil {
 		return err
 	}
-	return ep.assignAddressVersion(6, ipam)
+
+	err = ep.assignAddressVersion(6, ipam)
+	if err != nil {
+		return err
+	}
+
+	// Verify Ipam driver returned valid IP addresses for this endpoint
+	list := []*net.IPNet{ep.Iface().Address()}
```

Comment:
I guess the answer is over engineering ;-). 
Let me fix it following your suggestion.
Thanks